broker-core/src/main/java/io/zeebe/broker/workflow/processor/timer/TriggerTimerProcessor.java

```diff
@@ -18,35 +18,35 @@
 package io.zeebe.broker.workflow.processor.timer;
 
 import static io.zeebe.msgpack.value.DocumentValue.EMPTY_DOCUMENT;
+import static io.zeebe.util.buffer.BufferUtil.bufferAsString;
 
 import io.zeebe.broker.logstreams.processor.TypedRecord;
 import io.zeebe.broker.logstreams.processor.TypedRecordProcessor;
 import io.zeebe.broker.logstreams.processor.TypedResponseWriter;
 import io.zeebe.broker.logstreams.processor.TypedStreamWriter;
 import io.zeebe.broker.workflow.data.TimerRecord;
-import io.zeebe.broker.workflow.model.element.ExecutableBoundaryEvent;
-import io.zeebe.broker.workflow.processor.boundary.BoundaryEventActivator;
+import io.zeebe.broker.workflow.model.element.AbstractFlowElement;
+import io.zeebe.broker.workflow.model.element.ExecutableCatchEventElement;
+import io.zeebe.broker.workflow.processor.CatchEventBehavior;
+import io.zeebe.broker.workflow.state.DeployedWorkflow;
 import io.zeebe.broker.workflow.state.ElementInstance;
-import io.zeebe.broker.workflow.state.ElementInstanceState;
-import io.zeebe.broker.workflow.state.StoredRecord;
-import io.zeebe.broker.workflow.state.StoredRecord.Purpose;
 import io.zeebe.broker.workflow.state.TimerInstance;
 import io.zeebe.broker.workflow.state.WorkflowState;
 import io.zeebe.model.bpmn.util.time.RepeatingInterval;
 import io.zeebe.protocol.clientapi.RejectionType;
-import io.zeebe.protocol.impl.record.value.workflowinstance.WorkflowInstanceRecord;
 import io.zeebe.protocol.intent.TimerIntent;
-import io.zeebe.protocol.intent.WorkflowInstanceIntent;
 import io.zeebe.util.buffer.BufferUtil;
+import org.agrona.DirectBuffer;
 
 public class TriggerTimerProcessor implements TypedRecordProcessor<TimerRecord> {
-  private final BoundaryEventActivator boundaryEventActivator;
+
+  private final CatchEventBehavior catchEventOutput;
   private final WorkflowState workflowState;
 
   public TriggerTimerProcessor(
-      final WorkflowState workflowState, BoundaryEventActivator boundaryEventActivator) {
+      final WorkflowState workflowState, CatchEventBehavior catchEventOutput) {
     this.workflowState = workflowState;
-    this.boundaryEventActivator = boundaryEventActivator;
+    this.catchEventOutput = catchEventOutput;
```

Comment:
Sorry to be nitpicky :see_no_evil: but same with the field name. I thought in IntelliJ when you rename a class it'll ask if you want to rename a field with the same name :thinking:

Reply:
Yeah, I guess I forgot something :see_no_evil: